src/js/components/presentational/ImageList.jsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+// import PropTypes from "prop-types";
+
+// add functionality to change image on click/hover
+
+const ImageList = (props) => {
```

Comment:
I did some Googling and found a library called ImageSwipe that might be helpful here.

Reply:
Link: https://photoswipe.com/

Reply:
I did see that one, I am going to look into it more.